### PR TITLE
Initial action setup

### PR DESCRIPTION
**Pull request by fdosani**
First commit of the action and at housekeeping items.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  base-branch: # branch to base PR off
+    description: 'Branch to base PR off'
+    required: true
+    default: 'dev'
+runs:
+  using: "composite"
+  steps:
+    - run: echo Running edgetest with the following options ${{ inputs.edgetest-flags}}.
+      shell: bash
+    - uses: conda-incubator/setup-miniconda@v2
```
Comment (ak-gupta) [inline action.yml:17]: is there a similar mamba-based action? Might help speed things up for people.

Comment (fdosani) [reply action.yml:17]: I think it is experimental? https://github.com/conda-incubator/setup-miniconda#example-6-mamba

Comment (fdosani) [reply action.yml:17]: @ak-gupta maybe we can start with this and make mamba optional as a v2 item?

Comment (ak-gupta) [reply action.yml:17]: Sounds good!

**Review by ak-gupta — APPROVED** [review]

**Pull request merged**